templates/registry/markdown/event_macros.j2

```diff
@@ -1,4 +1,19 @@
 {#- Macros for simplifying creating "Event" documentation. -#}
-{% macro header(event) %}{% if event.name %}The event name MUST be `{{ event.name }}`.
+{% import 'stability.j2' as stability %}
+{% import 'body_field_table.j2' as body_table %}
+{% macro header(event) %}
 
+{% if event.name %}The event name MUST be `{{ event.name }}`.
+
+**Status:** {{ stability.badge(event.stability, event.deprecated) }}
```

Comment:
Nit: Similar to resource - Are we ok having attributes with different stability in a stable event?

Personally I think it's ok, but wonder if we should officially decide that as a group.

Reply:
let's keep discussing it in the https://github.com/open-telemetry/semantic-conventions/pull/1472 - this pr does not change the status quo.